web/packages/teleport/src/Clusters/ClusterList/ClusterList.tsx

```diff
@@ -84,6 +85,12 @@ function renderActionCell({ clusterId }: Cluster, flags: MenuFlags) {
     );
   }
 
+  $items.push(<DropdownDivider key="divider" />);
```

Comment:
not suggesting anything here, and I can see this is the way it has been done before, but I did not understand the reason for needing to do `items.push` to build cell items.